### DISCUSSION TITLE
How do you get data to show in the DetailsList?

### DISCUSSION THREAD

**Discussion by arpost1** [microsoft/powercat-creator-kit#109] [Q&A] [answered]
Greetings! I want to show data from a collection in the new DetailList component, but no matter what I do, I can't data or columns to show up. I've read all the guides I can find, but I still can't get data to show. Anyone have an idea/direction?

My collection has the columns "Title" and "Value".

**Comment by scottdurow** [comment, marked as answer]
Hi @arpost1, 
Let's say you had a collection that was initialized as :
```
ClearCollect(colItems, {Title:"row 1",Value:"value 1"},{Title:"row 2",Value:"value 2"});
```

There are 3 steps to get data to show up in the Fluent DetailsList:

1. Set the Items collection to `colItems`
2. Add the Title and Value columns using the **Edit** link below the **Items** dataset in the Control Properties side panel:
![image](https://user-images.githubusercontent.com/2871121/189726643-89408126-5be8-478e-93f9-532609ec314a.png)
This would result in the the fields panel looking similar to:
![image](https://user-images.githubusercontent.com/2871121/189727314-cf72f2d4-f1ef-4e34-b7f1-4332eb2faf3e.png)

3. Set the columns dataset to be something similar to:
```
Table({ColName:"Title",ColDisplayName:"Title",ColWidth:200},{ColName:"Value",ColDisplayName:"Value",ColWidth:200})
```

I suspect that maybe step 2 is the one that you are missing?

Hope this helps!
Scott

**Reply by arpost1** [reply]
That did it! Thanks, @scottdurow.
